bin/sort-pr-by-date.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
I think we should turn this into a bash (`.sh`) script instead, as we are basically running a single bash command.
If you feel up for it, I understand that writing it in php is probably easier.